st2common/bin/st2-run-pack-tests

```diff
@@ -65,14 +69,16 @@ STACKSTORM_VIRTUALENV_PYTHON_BINARY="${STACKSTORM_VIRTUALENV_BIN}/python"
 ####################
 
 function usage() {
-    echo "Usage: $0 [-x] [-j] [-v] -p <path to pack>" >&2
-    echo "  -x : Do not create virtualenv for test for tests, e.g. when running in existing one." >&2
-    echo "  -j : Just run tests (use previously installed dependencies" >&2
+    echo "Usage: $0  [-c] [-j] [-t] [-v] [-x] -p <path to pack>" >&2
+    echo "  -c : Run tests with code coverage reporting enabled" >&2
+    echo "  -j : Just run tests. Use previously installed dependencies" >&2
```

Comment:
Hmm. The old usage didn't quite get it right, but I think we can get it a bit better. Perhaps the following:

> Just run tests. Uses previously installed dependencies and virtualenv, if any, for subsequent test runs.